spec/features/logged_in_user_reserves_appointment_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+feature "logged in user reserves appointment" do
+  scenario "owning one car" do
+    user = create(:user)
+    car = create(:car, user: user)
+    login_as(user, as: :user)
+    visit dashboard_path
+
+    click_on "Solicitar turno"
+    expect(current_path).to eq(new_appointment_path)
+
+    fill_in "Seleccione una fecha", with: Date.tomorrow
+    fill_in "Seleccione un horario", with: Time.now.hour
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.